src/tables/AgTable.stories.tsx

```diff
@@ -73,3 +73,17 @@ export const Default = () => {
     />
   )
 }
+
+// The external library we are using is throwing an a11y error:
```

Comment:
Can you name the library here? that way we can potentially track the issue

Reply:
Updated!